package.json

```diff
@@ -49,45 +49,34 @@
     "url": "https://github.com/inrupt/solid-client-vc-js.git"
   },
   "devDependencies": {
-    "@inrupt/eslint-config-base": "^0.4.0",
-    "@inrupt/eslint-config-lib": "^0.4.0",
-    "@inrupt/solid-client-authn-node": "^1.11.2",
-    "@skypack/package-check": "^0.2.2",
+    "@inrupt/eslint-config-lib": "^1.0.0",
+    "@inrupt/solid-client-authn-node": "^1.12.1",
+    "@rushstack/eslint-patch": "^1.1.4",
     "@testing-library/testcafe": "^4.4.0",
     "@types/dotenv-flow": "^3.1.1",
     "@types/jest": "^27.0.1",
     "@types/node": "^17.0.6",
-    "@typescript-eslint/eslint-plugin": "^5.9.1",
-    "@typescript-eslint/parser": "^5.9.1",
     "dotenv-flow": "^3.2.0",
-    "eslint": "^7.32.0",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-header": "^3.1.1",
-    "eslint-plugin-import": "^2.24.0",
-    "eslint-plugin-jest": "^26.0.0",
-    "eslint-plugin-license-header": "^0.4.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "fast-check": "^2.17.0",
-    "jest": "^27.4.7",
+    "eslint": "^8.18.0",
```

Comment:
I think this eslint dep can also be removed - I was able to in the universal-fetch lib and the linters were still working.

Reply:
Similarly can license-checker and all of the license scripts go now that is part of the lint config.

Reply:
The reason for including it here is to ensure that the `npm bin` setup works correctly; between different systems the executables for node modules with bin directories can vary, so having it as a first-party dependency ensures it's always where we want it, rather than treating it as a transitive dependency.